bin/makeLinkFarm

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 
+from __future__ import print_function
```

Comment:
Please consider using our standard `from __future__ import absolute_import, division, print_function`. I admit division is unlikely in these files, but requiring absolute import is a useful addition, and it is the typical thing we add to all .py files.